nj-core/src/class.rs

```diff
@@ -137,7 +137,7 @@ pub trait JSClass: Sized {
             }
         })();
 
-        result.to_js(&js_env)
+        result.into_js(&js_env)
```

Comment:
why naming changes? I thought this PR was about clippy changes

Reply:
One of the clippy lints is making sure that methods have conventional names based on their `self` type. Methods that take `self` by ownership should have the form `into_X`. `to_X` is used for `&self`.

Reply:
I also checked: this trait is not exposed to the outside of this crate, so this is not a breaking change.